### ISSUE TITLE
Unhelpful hover-over text for added links

### ISSUE THREAD

**Issue by Aevaris** [manyfold3d/manyfold#3619]
**Describe the bug**
If you mouse over a link, you get a 'translation missing en.sites.hostnameOfLink' message. Unsure if this is a manyfold problem or a source website problem, but in either case the hover-over text isn't super helpful for the user so should either be a more helpful message or not be shown.

![Image](https://github.com/user-attachments/assets/a832c99e-92a1-4417-a808-f2db1ecaeec1)

URL in this image:
https://bulbapedia.bulbagarden.net/wiki/Mew_(Pok%C3%A9mon)

**Comment by Floppy** [comment]
Ooh, that's definitely a bug. Thank you!
